CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## Upgrade Notes
 
+- **Bump Ruby to v2.6**
+
+As per [\#6320](https://github.com/decidim/decidim/pull/6320) we've bumped the minimum Ruby version to 2.6.x.
```

Comment:
Why not specify 2.6.6?

Reply:
I thought the same while you were writing XD